.drone.yml

```diff
@@ -281,12 +281,23 @@ pipeline:
       event: [ push ]
       branch: [ master ]
 
+  docker-dryrun:
+    image: plugins/docker:17.12
+    secrets: [ docker_username, docker_password ]
```

Comment:
why does dry run needs docker username and password?

Reply:
@lafriks removed.